rpc/amqp-rpc.go

```diff
@@ -353,7 +353,11 @@ func (rpc *AmqpRPCServer) processMessage(msg amqp.Delivery) {
 		rpc.log.Audit(fmt.Sprintf(" [s>][%s][%s] Error condition marshalling RPC response %s [%s]", rpc.serverQueue, msg.ReplyTo, msg.Type, msg.CorrelationId))
 		return
 	}
-	rpc.log.Info(fmt.Sprintf(" [s>][%s][%s] replying %s(%s) [%s]", rpc.serverQueue, msg.ReplyTo, msg.Type, core.B64enc(jsonResponse), msg.CorrelationId))
+	if response.Error.Value != "" {
+		rpc.log.Info(fmt.Sprintf(" [s>][%s][%s] %s failed, replying: %s [%s]", rpc.serverQueue, msg.ReplyTo, msg.Type, response.Error.Value, msg.CorrelationId))
+	} else {
+		rpc.log.Info(fmt.Sprintf(" [s>][%s][%s] replying %s(%s) [%s]", rpc.serverQueue, msg.ReplyTo, msg.Type, core.B64enc(jsonResponse), msg.CorrelationId))
```

Comment:
Or we could reduce this whole log line to Debug log level, which I assume will not be collected in prod? I think we probably don't need to log every RPC response at Info, right?